### ISSUE TITLE
Paragraph "After" is not working anymore?

### ISSUE THREAD

**Issue by NatashaDK** [photopea/photopea#186]
Text distance between 2 lines, in the Paragraph Panel, don't seem to work anymore 👍 

![after is not working](https://cloud.githubusercontent.com/assets/27579710/26072840/52445cd8-39ad-11e7-8de7-9cd38ac3458b.png)

**Comment by NatashaDK** [comment]
Sorry 'bout this... I'm using LEADING now :-)

**Comment by photopea** [comment]
Hi Natasha ... Before and After represent spaces between Paragraphs. I.e. you must physically hit enter (insert a "newline character") after the first part of text, to make the second part a new paragraph.

If you wrote BEIGE SILK SCARF $30 without making a new line after SILK, it is considered the same paragraph. You should use Leading to control gaps between lines within a single paragraph.

**Comment by NatashaDK** [comment]
You are absolutely right. I zoomed the text to 2 lines. And I was half asleep at the time. 
I testet it again today and everything works perfect. I'm sorry about this posting... it should never have been made. 
And I do know about a "normal Return" and a shift + Return". Or a "Shift-Go" as we call it here in Denmark. Kudos to you for implementing these character tricks in a very easy manner inside PP. In fact text handling is easier in PP than in Microsoft Word :-)

Love from Natasha / Denmark

**Comment by photopea** [comment]
Ther is no need to be sorry :) I really apreciate that you report so many problems.

**Comment by NatashaDK** [comment]
Thanx, I will !
